src/main/resources/db/migrationv2/V2_43__housekeeping_trial_deletion_update.sql

```diff
@@ -0,0 +1,77 @@
+-- DROP PROCEDURE juror_mod.housekeeping_trial_deletion(in int4, in int4, in int4, inout text);
+
+CREATE OR REPLACE PROCEDURE juror_mod.housekeeping_trial_deletion(IN p_max_threshold integer, IN p_start_time_int integer, IN p_max_timeout integer, INOUT p_print_msg text)
+ LANGUAGE plpgsql
+AS $procedure$
+DECLARE
+   	v_text_var1 TEXT;
+   	v_text_var2 TEXT;
+   	v_text_var3 TEXT;
+   	v_timed_out BOOLEAN;
+
+BEGIN
+	<<Deletes>>
+	BEGIN
+		-- child records
+		WITH aged_trials AS
+		(
+			SELECT t.trial_number
+			FROM juror_mod.trial t
+			WHERE t.trial_end_date < CURRENT_DATE - p_max_threshold
+		), excluded_trials AS
+    (
+      SELECT jt.trial_number
```

Comment:
Need to select loc_code as well as trial_no

Reply:
addressed in code update

Reply:
loc_code has been added to aged_trials but not excluded_trials. Don't we also need to include it in excluded_trials otherwise we could be excluding trials based on data from other courts?

Reply:
My bad. We don't need to include loc_code in excluded_trials. All okay as is.